README.md

```diff
@@ -1 +1,3 @@
 # 499TeamExercise
+
```

Comment:
Modifying README.md in your own branch is good practice that we should all have done, koodos for that. Because some of us (including me) have modified READ.md directly from master you will have merge conflicts so make sure you pull README.md onto your branch to resolve those conflicts